addons/sale_purchase/models/sale_order.py

```diff
@@ -288,7 +288,10 @@ def _purchase_service_create(self, quantity=False):
                 ], limit=1)
             if not purchase_order:
                 values = line._purchase_service_prepare_order_values(supplierinfo)
-                purchase_order = PurchaseOrder.create(values)
+                if self.env.user.has_group('base.group_user'):
+                    purchase_order = PurchaseOrder.create(values)
+                else:
+                    purchase_order = PurchaseOrder.with_context({'mail_create_nosubscribe': True}).create(values)
```

Comment:
I don't think we should still allowing internal users to follow the PO in this case. Imagine when a salesman creates a SO and then he can read the whole conversation between purchasing staff and vendor on the PO.